include/stage_ros2/stage_node.hpp

```diff
@@ -24,6 +24,9 @@
 #include <stage_ros2/static_transform_broadcaster.h>
 #include <tf2/transform_datatypes.h>
 #include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <stage_ros2/srv/set_object_pose.hpp>
+#include <visualization_msgs/msg/marker.hpp>
```

Comment:
You can move the following two includes into the cpp since you do not use them here.